code/modules/mob/living/silicon/death.dm

```diff
@@ -6,3 +6,9 @@
 
 /mob/living/silicon/spawn_dust()
 	new /obj/effect/decal/remains/robot(loc)
+
+/mob/living/silicon/death(gibbed)
+	diag_hud_set_status()
+	diag_hud_set_health()
+	update_health_hud()
+	..()
```

Comment:
not specifying any argument inside `..()` means it will use the arguments of the current proc. You only specify arguments inside ..() if you want it to be called with different arguments than the default.